src/basic/virt.c

```diff
@@ -899,6 +899,12 @@ int running_in_chroot(void) {
         if (getenv_bool("SYSTEMD_IGNORE_CHROOT") > 0)
                 return 0;
 
+        r = getenv_bool("SYSTEMD_IN_CHROOT");
```

Comment:
Hmm, wouldn't `SYSTEMD_OFFLINE=0` satisfy your need? Maybe instead make that generic and switch some running_in_chroot() detections to `_or_offline()`?

Reply:
I think `SYSTEMD_IGNORE_CHROOT` and `SYSTEMD_OFFLINE` are distinct for a reason and we should keep it that way.